nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -126,6 +127,7 @@ def _apply(
     ) -> TModel:
         self._set_backend_entity(model)
 
+        model_copy = self._backend_entity.remove_fq_from_inputs(copy_model(model))
```

Comment:
You can't control the `presence of an activation FQ`, because you need to analyze all inputs and keep in mind that FQ can be represented as Q-DQ pair. Also, there is no clear difference between FQ on weights and on activations. To determine a type of FQ node, you need to propagate through its inputs and check out the input type and keep in mind that weights can be represented as a `Constant -> Convert` subgraph and other variations.